workspaces/arborist/lib/arborist/reify.js

```diff
@@ -138,6 +140,10 @@ module.exports = cls => class Reifier extends cls {
 
   // public method
   async reify (options = {}) {
+    const linked = (
+      options.installStrategy === 'linked'
+      || this.options.installStrategy === 'linked')
```

Comment:
if the options explicitly override the installStrategy, shouldn't that take precedence over `this.options`?

meaning:
```suggestion
      (options.installStrategy || this.options.installStrategy) === 'linked'
```